src/EA.Weee.RequestHandlers/Organisations/OrganisationByIdHandler.cs

```diff
@@ -133,8 +122,7 @@ private async Task SetDirectRegistrants(OrganisationData organisationData, Guid
                     YearSubmissionStarted = directRegistrant.HasCurrentYearSubmission,
                     RepresentedCompanyName = directRegistrant.AuthorisedRepresentativeId.HasValue
                         ? directRegistrant.OverseasProducerName
-                        : null,
-                    MostRecentSubmittedYear = directRegistrant.MostRecentSubmittedYear
```

Comment:
No longer required as we just pass -1 from the Choose Activity to allow the registrations to select the most recent available year